### DISCUSSION TITLE
Unique class added to differentiate INTERNAL and EXTERNAL links

### DISCUSSION THREAD

**Discussion by ReessKennedy** [outline/outline#4314] [Ideas]
Can a simple class be added to internal links like "internal" (or something similar) as currently the classes for both internal and external seem the same. 

Normally this might be okay but I have been customizing the CSS for my install to great effect and one thing I'd like to be able to do is make internal links stand out more but keep external links the way they are. To do this, internal links would need to have a unique class. 

<img width="619" alt="Screen Shot 2022-10-17 at 11 27 52 PM" src="https://user-images.githubusercontent.com/3970267/196328997-80e0b285-5986-42b0-8308-040001e16d39.png">

**Comment by tommoor** [comment]
I think you can use a css selector to target `[href^="http"]`, it would be more reliable

**Reply by ReessKennedy** [reply]
Interesting! Thanks @tommoor
